plugins/modules/helm_cli_info.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: helm_cli_info
```

Comment:
Qq: Do we want to just call these `helm` and `helm_info` (instead of `helm_cli` and `helm_cli_info`)? Since this is entirely new content in the collection we don't have to worry about any old helm module...

Reply:
The one thing to consider against this would be forcing the use of the FQN of the module to avoid the helm module that is in core today and ACD in the future. Just a thought.

Reply:
It's ok for me
I already made a PR on ACD to put helm module in deprecated https://github.com/ansible-collections/community.general/pull/115
It will be removed in 4 releases